package/base-files/files/lib/upgrade/common.sh

```diff
@@ -23,10 +23,25 @@ install_bin() { # <file> [ <symlink> ... ]
 	install_file $files
 	shift
 	for link in "$@"; do {
-		dest="$RAM_ROOT/$link"
-		dir="$(dirname $dest)"
-		mkdir -p "$dir"
-		[ -f "$dest" ] || ln -s $src $dest
+		# Handle the case of using full versions of
+		# commands instead of busybox version; this
+		# only matters when busybox version is not
+		# compiled in (and hence /{s,}bin link doesn't
+		# exist)
+		if [ -L "$link" ]; then
```

Comment:
It's because it's for use with the above busybox option which make busybox not include the normal busybox applets and instead selects packages (which packages depends on which options you enable, and what packages are available as replacements in your build) with full versions.